Portfolio1.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<!--For the tab and CSS document link -->
+<head>
+	<title>Portfolio Webpage</title>
+	<link rel="stylesheet" type="text/css" href="portfolio1.css">
+</head>
+
+<body>
+
+<!--Top Bar Div: for Logo, Skip-to-Main-Text and Sign-In Form-->
+<div class="top-bar">	
+
+	<!--Linked Logo Top Left -->
+	<a target="_blank" href="https://www.history.com/topics/ancient-history/tutankhamen"> 
+		<img src="pexels-pixabay-33571.jpg" alt="Gold plated crest from the sarcophagi of Tutankharmun, with a link to the history of the pharoh" class="logo-link">
+	</a> 
+
+	<!--Skip-to Button -->
+	<div class="accessibility-link">
+		<a href="#text-beginning">Skip to Main Text</a>
+	</div>
+
+	<!--Sign-in Form -->
+	<form class="form">
+		<!--Email -->
+		<label for="email">Email</label>
+		<input type="email" name="email" id="email" />
+
+		<!--Password -->
+		<label for="password">Password</label>
+		<input type="password" name="password" id="password" />
+		
+		<!--Checkbox -->
+		<label for="remember">Remember Me</label>
+		<input type="checkbox" name="remember" id="remember" />
+
+		<!--Submit button -->
+		<input type="submit" value="Submit" id="submit"/>
+	</form>
+
+</div>
+
+<!--Buttons and Search Bar-->
+<!--Buttons -->
+<div class="buttons1">
+	<button>HOME</button>
+	<button>ABOUT</button>
+	<button>LOCATIONS</button>
+	<button>NEWS</button>
+	<button>STAFF</button>
+	<button>CONTACT</button>
+<!--Search Bar -->
+	<input type="text" placeholder="Search" name="search"/>
+	<input type="submit" name="search" value="Search" id="submit"/>
+</div>
+
+<!--Central Image and Heading1 -->
+<div class="main-image-div">
+	<img src="pexels-pixabay-260024.jpg" alt="A lone chesspiece stands between the two sides 	wearing a golden crown" class="main-image">
```

Comment:
in your alt you have a big gap

Reply:
fixed it :)

Reply:
oh... I still see it...
"A lone chesspiece stands between the two sides 	wearing a golden crown" 
I meant the big gap after the word sides...

Reply:
Unless is github. Sorry...